source/attention_seekers/pulse.css

```diff
@@ -1,19 +1,25 @@
 /* originally authored by Nick Pettit - https://github.com/nickpettit/glide */
-
+/* rewrote by Procodeing - https://github.com/procodeing */
```

Comment:
This is hardly a rewrite; credit where credit's due, but the changes here are minor. You'll still be included in the contributors page on the repo if/when this is merged!